apps/gitness/src/pages-v2/pull-request/hooks/usePRCommonInteractions.ts

```diff
@@ -50,8 +53,9 @@ export function usePRCommonInteractions({
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     fileBlob: any
   ) => {
+    const apiPath = `${baseAPIPath}/${repoRef}/uploads`
     try {
-      const response = await fetch(`${window.location.origin}${`/api/v1/repos/${repoRef}/uploads`}`, {
+      const response = await fetch(`${window.location.origin}${apiPath}`, {
```

Comment:
why have two `useApiPath` functions if we're still hardcoding string concatenation with `window.location` 😅 let's discuss